README.md

```diff
@@ -37,6 +37,15 @@ Make use of the many generators for code, try `ember help generate` for more det
 * `ember test`
 * `ember test --server`
 
+### Browserstack testing
```

Comment:
I can't get the tests running locally. It's not getting the keys from my `.env` file.

Reply:
The tests should run fine locally `ember test`.

Reply:
I'm gonna see about getting it to pick up the `.env` file properly